### ISSUE TITLE
Final demand biodiesel for transport correction

### ISSUE THREAD

**Issue by lottevanvlimmeren** [quintel/etsource#2568]
The demand for node `transport_final_demand_biodiesel` is determined directly from the energy balance as following:

![image](https://user-images.githubusercontent.com/89903998/139683272-3499ba8b-7dda-4173-820f-35e269867bf9.png)

For the "transport" flow, the products "biodiesels", "other liquefied biofuels", and "non-specified primary biofuels and waste" are being used. For the "non-specified transport" and "pipeline transport" flows, the product "biodiesels" is determined from the energy balance three times. I assume this is a copy-paste mistake, and that the two of the three "biodiesels" could be changed into "other liquefied biofuels" and "non-specified primary biofuels". @mabijkerk do you agree?

**Comment by mabijkerk** [comment]
I agree that it is probably a copy-paste mistake. The `8_transport_analysis` clearly shows that we assume the ETM carrier 'biodiesel' to consist of 'biodiesels', 'other liquid biofuels' and 'non-specified primary biofuels and waste'.

Could you check other transport nodes for similar mistakes and update them to the correct products? I checked `transport_final_demand_gasoline.final_demand.ad` and it contains a similar mistake for 'motor gasoline'.

**Comment by lottevanvlimmeren** [comment]
The demand for node `transport_final_demand_biodiesel` seems to have been rewritten already. There were two other nodes that had the same mistake: `transport_final_demand_coal` and `transport_final_demand_gasoline`.
